packages/eslint-plugin-query/src/rules/exhaustive-deps.test.ts

```diff
@@ -0,0 +1,83 @@
+import { ESLintUtils } from '@typescript-eslint/utils'
+import { exhaustiveDepsRule } from './exhaustive-deps.rule'
+
+const ruleTester = new ESLintUtils.RuleTester({
+  parser: '@typescript-eslint/parser',
+  settings: {},
+})
+
+ruleTester.run('exhaustive-deps', exhaustiveDepsRule, {
+  valid: [
```

Comment:
can we add a test that just shows that a basic useQuery is okay?

```
useQuery(['todos'], fetchTodos)
```

Reply:
added as an "object" syntax